src/Juvix/Compiler/Pipeline/Loader/PathResolver/Data.hs

```diff
@@ -7,6 +7,7 @@ import Juvix.Prelude
 
 data ResolverEnv = ResolverEnv
   { _envRoot :: Path Abs Dir,
+    _envProjectRoot :: Path Abs Dir,
```

Comment:
we should clarify in a comment what is the difference between the two roots

Reply:
Done. Added comment and renamed to `envInitialRoot`.